frontend/src/utils/fileUtils/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./createFormData";
```

Comment:
we're not using this type of pattern at this point. For now can you reference files directly and we can discuss a change?